### PR TITLE
fix: update coverage_history_dml

### PR DESCRIPTION
**Pull request by yjhuoh**
was marking history as invalidated when a group contained an invalidated user_data and the current one

### CODE REVIEW

**Comment by yjhuoh** [comment]
after running:
` insert into user_data_override(user_id, field, override, created_at) values (21, 'coverage_level', 'awesome', '2017-05-01'::timestamp);`
and 
```
update user_data set invalidated_at = '2017-04-30'::date where user_id = 28;
insert into user_data(user_id, date_of_birth, first_name, last_name, start_date, coverage_level) values (28, '1988-05-06'::date, 'first', 'last', '2017-04-29'::date, 'individual');
```

screen shot 2017-06-16 at 13 31 31

File: dmls_to_run/populate_coverage_history.sql [hunk 1]
```diff
@@ -2,8 +2,15 @@ INSERT INTO coverage_histories (
...
 )
-SELECT user_id, start_date, end_date, coverage_level, MAX(invalidated_at),
-  MIN(created_at), MAX(updated_at)
+SELECT
+  user_id
+  , start_date
+  , end_date
+  , coverage_level
+  , NULLIF(MAX(COALESCE(invalidated_at, '2099-12-31'::timestamptz)),
+           '2099-12-31'::date)
```
Comment (maxsmithrx) [inline dmls_to_run/populate_coverage_history.sql]: Why timestamptz on first line and date on 2nd?  Looks good to me other than that

Comment (yjhuoh) [inline dmls_to_run/populate_coverage_history.sql]: good catch - i was typing lazily

**Comment by maxsmithrx** [comment]
👍

**Pull request merged**